cctp_primitives/Cargo.toml

```diff
@@ -33,7 +33,7 @@ bzip2 = { version = "=0.4.4", features = ["static"] }
 flate2 = "=1.0.21"
 
 [dev-dependencies]
-serial_test = "*"
+serial_test = "0.5.1"
```

Comment:
```suggestion
serial_test = "=0.5.1"
```
?

Reply:
If you prefer... but there isn't any other 0.5.x version (the newer version is 1.0.0). That will not change anything

Reply:
Done